ghost/i18n/locales/ca/portal.json

```diff
@@ -46,166 +46,166 @@
+    "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Si un butlletí es marca com a correu brossa, els correus electrònics es desactivaran automàticament per a aquesta adreça per assegurar-te que ja no reps cap missatge no desitjat.",
+    "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Si la queixa de correu brossa va ser accidental, o si vols començar a rebre correus electrònics de nou, pots tornar a subscriure't als correus electrònics fent clic al botó de la pantalla anterior.",
+    "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Si cancel·les la teva subscripció ara, continuaràs tenint-hi accés fins al {{periodEnd}}.",
+    "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Si tens un compte de correu electrònic corporatiu o governamental, posa't en contacte amb el vostre departament de TI i demana'ls-hi que permetin rebre correus electrònics de {{senderEmail}}",
+    "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Si vols començar a rebre correus electrònics de nou, el millor que pots fer és comprovar si hi ha cap problema a la teva adreça de correu electrònic i, a continuació, fes clic a tornar a subscriure's a la pantalla anterior.",
+    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Si no reps el butlletí electrònic al qual t'has subscrit, aquí tens algunes coses per comprovar.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Si has completat totes aquestes comprovacions i encara no reps correus electrònics, pots contactar amb {{supportAddress}} per obtenir assistència.",
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "En cas que es rebi un error permanent en intentar enviar un butlletí, els correus electrònics es desactivaran en aquest compte.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Afegeix {{senderEmail}} a la llista de contactes del teu client de correu electrònic. Això indica al teu proveïdor de correu que els correus electrònics enviats des d'aquesta adreça haurien de ser de confiança.",
+    "Invalid email address": "Adreça de correu electrònic invàlida",
+    "Jamie Larson": "Jamie Larson",
```

Comment:
Jamie Larson is intended to be a placeholder name.  Feel free to suggest a name that would be recognizable to Catalan speakers.

Reply:
Awesome, will do it.